src/dask_dirac/_dask.py

```diff
@@ -32,69 +41,44 @@ def __init__(
         scheduler: Any = None,
         name: str | None = None,
         config_name: str | None = None,
-        submission_url: str | None = None,
-        user_proxy: str | None = None,
-        cert_path: str | None = None,
-        jdl_file: str | None = None,
-        owner_group: str | None = None,
-        dirac_site: str | None = None,
+        submission_url: str = "https://lbcertifdirac70.cern.ch:8443",
```

Comment:
Should probably make the dev server the default given that we use that exclusively now: https://diracdev.grid.hep.ph.ic.ac.uk:8444